pkg/cloudprovider/controller/nodecontroller.go

```diff
@@ -437,6 +437,17 @@ func (s *NodeController) deletePods(nodeID string) error {
 		if err := s.kubeClient.Pods(pod.Namespace).Delete(pod.Name); err != nil {
 			glog.Errorf("Error deleting pod %v: %v", pod.Name, err)
 		}
+		if pod.Spec.RestartPolicy == api.RestartPolicyAlways {
```

Comment:
@bgrant0607 

Is the expectation that all pods are started with replication controller? Even if replicas is 1? Would it make sense to identify pods that are started by replication controller (may be through an annotation - I dont know of any other way) and skip them?